woocommerce/wc-admin
# Add new note for the theme marketplace

Fixes #5955 

This PR adds a new note about the theme marketplace.


### Screenshots
![Screen Shot 2021-01-05 at 6 23 17 PM](https://user-images.githubusercontent.com/4723145/103721777-693f5380-4f83-11eb-81cd-f64c8dfee162.jpg)

### Detailed test instructions:

1. Navigate to WordPress -> Plugins
2. Deactivate WooCommerce Admin and activate it again
3. Open `wp_wc_admin_notes` table and confirm a new note with name `wc-admin-choosing-a-theme`
4. Navigate to WooCommerce -> Home and confirm that you have a new note with the title `Choosing a theme?` (refer to the screenshot)
